Formula/newsboat.rb

```diff
@@ -5,12 +5,7 @@ class Newsboat < Formula
   sha256 "82d5e3e2a6dab845aac0bf72580f46c2756375d49214905a627284e5bc32e327"
   head "https://github.com/newsboat/newsboat.git"
 
-  bottle do
-    sha256 "9dcbf0d0a026bc0c0667ce52698365f2f172e1222334657a4327c1101304b8ff" => :high_sierra
-    sha256 "2094f90b7a64c045e408973420cec64805f4ec517a9397f5850d35d9c013151e" => :sierra
-    sha256 "9af1f9da45dcb359481827c3e61e2aef3e21c253b29d436962369b8816be25cc" => :el_capitan
-  end
-
+  depends_on "asciidoc" => :build
```

Comment:
if this is needed for head builds only then you need a `head do` block
```
  head do
    url "https://github.com/newsboat/newsboat.git"

    depends_on "asciidoc" => :build
  end
```

Reply:
I'm not sure it's for head builds only, I'll double check